internal/services/device_definition_service_mock_test.go

```diff
@@ -0,0 +1,248 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
I see. You get an import cycle. I would like to stop dumping everything in `/services` anyway. Let's revisit in a bit.